mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -265,6 +266,18 @@ class ExperimentView extends Component {
     const compareDisabled = Object.keys(this.state.runsSelected).length < 2;
     const deleteDisabled = Object.keys(this.state.runsSelected).length < 1;
     const restoreDisabled = Object.keys(this.state.runsSelected).length < 1;
+    const searchInputHelpTooltipContent = (
+      <div className="search-input-tooltip-content">
+        Click here to learn more about the search syntax.<br/>
+        <a
+          href="https://mlflow.org/docs/latest/tracking.html"
```

Comment:
Btw, importantly, do we currently document our search syntax @mparkhe? Do we have a place that we plan to document it?

Reply:
Yeah, we should link to a location within that page eventually if possible.

Reply:
👍 
cc: @dbczumar